mercury.spec

```
@@ -26,7 +26,7 @@ Group:    Development/Libraries
 License:  Argonne National Laboratory, Department of Energy License
 URL:      http://mercury-hpc.github.io/documentation/
 Source0:  https://github.com/mercury-hpc/mercury/archive/v%{dl_version}.tar.gz
-Patch0:   https://github.com/daos-stack/mercury/na_ucx_local_addr.patch
+Patch0:   https://patch-diff.githubusercontent.com/raw/mercury-hpc/mercury/pull/617.patch
```

This is still an unstable link that could disappear when that upstream PR is merged and/or garbage collected at some point in the future I think.

I think @soumagne's [previous suggestion](https://github.com/daos-stack/mercury/pull/85#discussion_r967242259) is probably the best way to go here.  Generally speaking patches are referred to in specfiles without a URL as they are included in the build sources.  That we have been specifying URLs for them is actually exceptional (in the meaning off not normal), not following usual standards and processes for specfiles.

```suggestion
Patch0:  na_ucx_local_addr.patch
```

and resurrect that `na_ucx_local_addr.patch` file in this repo.